src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final int BOUND = 100;
+    private final String white = "WHITE";
+    private final int radius = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public int getRandomNum() {
+        return random.nextInt(BOUND);
+    }
+
+    public String getRandomFigure() {
+        return FigureList.values()[random.nextInt(FigureList.values().length)].name();
+    }
+
+    public Figure generateFigure(Enum Figure) {
+        switch (Figure.name()) {
+            case ("CIRCLE"):
```

Comment:
perfect opportunity to use your enum here

Reply:
agree with Oleg